rust/feature-flags/src/api/errors.rs

```diff
@@ -13,6 +13,8 @@ pub enum ClientFacingError {
     Unauthorized(String),
     #[error("Rate limited")]
     RateLimited,
+    #[error("billing limit reached")]
+    BillingLimit,
```

Comment:
will implement the quota limiting in another PR; I started on it and then it got kinda beefy.  Leaving this in for now though since it's harmless to add the error type, the PR will land shortly anyway.